src/shared/js/components/Sidekick/ElementHighlighter.js

```diff
@@ -242,7 +247,7 @@ const SectionUI = ({ pageId, section, setAddToTemplate, index, active }) => {
   return (
     <SectionUIContainer
       className="csk-section"
-      active={active}
+      $active={active}
```

Comment:
Typo?

Reply:
I was trying to resolve an issue with styledcomponents, but it did not work. I think I removed this in the other PR